index.html

```diff
@@ -264,103 +264,96 @@ <h2>Intro Section of the Document</h2>
-            <ul>
-              <li>Examples: <code>&quot;subprotocol&quot;:&quot;sse&quot;</code>, <code>&quot;href&quot;:&quot;http://example.com&quot;</code>, <code>&quot;contentType&quot;:&quot;application/json&quot;</code></li>
-              <li>In a TD, a binding SHOULD be identifiable by the terms in a form such as <code>href</code>, <code>contentType</code>, <code>subprotocol</code>, or connection information found in the root of the TD.<ul>
-              <li>Reasons Behind the Requirement:<ul>
-              <li>This avoids conflicts that are mentioned in the previous requirement</li>
-              </ul>
-              </li>
-              <li>TODO: These terms should be refined based on the additions/changes to the TD 2.0 mechanism, e.g., introducing a <code>protocol</code> term or putting restrictions on URI scheme and <code>subprotocol</code> combination, data mapping, etc.</li>
-              <li>Until a custodian review, no registration is needed. A full IANA registration is required for the final and stable version of the binding. The submitter SHOULD trigger the registration at IANA. If needed, the custodian MAY trigger the IANA registration. The submitter MAY do a provisional registration to simplify the process on the IANA side.</li>
-              </ul>
-              </li>
-            </ul>
-          </dd>
-          <dt>
-            <dfn id="form-tdver">Form-TdVer</dfn>
-          </dt>
-          <dd>
-            <p>Supported TD version (no uniqueness needed):</p>
-            <ul>
-              <li>A binding SHOULD correspond to specific TD specification version(s).<ul>
-              <li>Reason Behind the Requirement: A binding may not fit newer or older versions of a TD specification (e.g., <code>readproperty</code> can become <code>readprop</code>, or a new operation can arrive). Thus, when writing a binding, it must be associated with one or more known TD specification versions.</li>
-              </ul>
-              </li>
-            </ul>
-          </dd>
-          <dt>
-            <dfn id="form-stat">Form-Stat</dfn>
-          </dt>
-          <dd>
-            <p>Status: One of Initial, Current, Superseded or Obsolete</p>
-          </dd>
-          <dt>
-            <dfn id="form-summ">Form-Summ</dfn>
-          </dt>
-          <dd>
-            <p>Summary Document: Link to the summary document</p>
-          </dd>
-          <dt>
-            <dfn id="form-ver">Form-Ver</dfn>
-          </dt>
-          <dd>
-            <p>Version: A unique string for that entry&#39;s history that denotes the version of the entry that is linked. The version string SHOULD contain a UTC-based date in ISO 8601 format in the form of <code>YYYY-MM-DD</code>.</p>
-          </dd>
-        </dl>
+        <p>Each entry MUST contain the following information, and all parts of the entry MUST not conflict with existing bindings.</p>
```

Comment:
some information like status will conflict but that is obvious. Maybe the table below can have a `unique: true/false`?